dashboard-api/internal/testhelpers/testhelpers.go

```diff
@@ -136,12 +137,20 @@ func DeleteAllUsers() {
 
 func CreateStudy(slug, userID string) error {
 	r := GetRepositories()
-	q := "INSERT INTO studies (slug, name, user_id) VALUES ($1, $2, $3)"
-	_, err := r.Db.Exec(q, slug, slug, userID)
+	q := "INSERT INTO studies (id, slug, name, user_id) VALUES ($1, $2, $3, $4)"
+	_, err := r.Db.Exec(q, slug, slug, slug, userID)
 	return err
 }
 
 func CreateUser() {
 	r := GetRepositories()
 	_, _ = r.Db.Exec("INSERT INTO users (id) VALUES ($1)", CurrentUserId)
 }
+
+func CreateDatabaseStudyConf(dsc types.DatabaseStudyConf) error {
+
+	r := GetRepositories()
```

Comment:
excuse my silly Go question: is that `r` for respositories and `q` for query?

Reply:
yes, this is definitely not one of my favorite practices in Golang, but they tend to push towards very simple variables names that understanding can be pulled from the context of the decleration, in this case:
```golang
// We can assume that r is now a repositories structure due to the naming of the function
r  :=  GetRepositories()
```
There is quite a large [styleguide](https://google.github.io/styleguide/go/best-practices#variable-declarations) for golang written by google: